README.md

```diff
@@ -6,9 +6,12 @@
 [![Platform](https://img.shields.io/cocoapods/p/Alamofire.svg?style=flat)](https://alamofire.github.io/Alamofire)
 [![Twitter](https://img.shields.io/badge/twitter-@AlamofireSF-blue.svg?style=flat)](https://twitter.com/AlamofireSF)
 [![Gitter](https://badges.gitter.im/Alamofire/Alamofire.svg)](https://gitter.im/Alamofire/Alamofire?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+[![Open Source Helpers](https://www.codetriage.com/alamofire/alamofire/badges/users.svg)](https://www.codetriage.com/alamofire/alamofire)
 
 Alamofire is an HTTP networking library written in Swift.
 
+**WARNING** This documentation is out of date during the Alamofire 5 beta process.
```

Comment:
Good call. Might even be worth throwing some ⚠️⚠️⚠️ characters in there to try to draw more visual attention to this statement.

Reply:
Added warning signs.